src/transform.ts

```diff
@@ -146,12 +143,12 @@ export interface WindowTransform {
   ignorePeers?: boolean;
 
   /**
-   * The fields to group by.
+   * The names of the data fields to partioin the objects into seprate windows. If not specified, everything will be in a single group.
    */
   groupby?: string[];
 
   /**
-   * The definitions of how to sort each of the fields in the window.
+   * A definition for sorting the objects within the window. Equivalent objects are considered a peer (Look at ignorePeers). If left undefined, the order of items in the window is undefined.
```

Comment:
A definition for sorting the data objects  (add data)

if left undefined => If undefined